Tools/validate.py

```
@@ -0,0 +1,78 @@
+#! /usr/bin/env python3
+""" Script to validate YAML file(s) against a YAML schema file """
+
+import argparse
+import logging
+import pprint
+import json
+import sys
+import os
+
+import yaml
+import cerberus
+
+
+def load_data_file(file_name: str, filetype: str):
+    with open(file_name) as stream:
+        if filetype == "yaml":
+            return yaml.safe_load(stream)
+        elif filetype == "json":
+            return json.load(stream)
+        else:
+            return dict()
```

Better to raise an exception here